apps/question-the-docs/backend/ai/utils/github.py

```diff
@@ -1,12 +1,32 @@
 "AI helper functions for loading data from GitHub."
 
 import base64
-import json
 import os
 from pathlib import Path
 
 import requests
 
+REPOS = {
+    'superduperdb': {
```

Comment:
An impressive list!!

But... shouldn't there be some dataclass called `Repo`, and then the correctness of these fields gets checked early?